bindings/tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms of the
+# BSD-3-Clause license. See the accompanying LICENSE file for details.
+
+macro(add_python_unit_test pythonscript)
+  if(PYTHON_EXECUTABLE)
+    add_test(NAME ${pythonscript} COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/${pythonscript})
+    set_tests_properties(${pythonscript} PROPERTIES ENVIRONMENT "PYTHONPATH=$<TARGET_FILE_DIR:${SWIG_MODULE_icub_python_REAL_NAME}>")
+  endif()
+endmacro()
+
+file(GLOB icub_python_test_files "${CMAKE_CURRENT_SOURCE_DIR}/*.py")
```

Comment:
Is that being used?

Reply:
Actually, I only follow the same way in yarp [tests CMake](https://github.com/robotology/yarp/blob/master/bindings/python/tests/CMakeLists.txt). This requires some more changes in https://github.com/robotology/icub-main/pull/541 to really effect